NAMESPACE

```diff
@@ -1,8 +1,10 @@
 # Generated by roxygen2: do not edit by hand
 
 S3method(print,tidyomics_conflicts)
+export(tidyomics)
 export(tidyomics_conflicts)
 export(tidyomics_packages)
+importFrom(BiocManager,install)
```

Comment:
are we installing something sutomatically?

Reply:
if this is used in the read me, this should not be needed

Reply:
I have removed the BiocManager import